definitions/output/reports/tech_report_lighthouse.js

```diff
@@ -63,11 +65,11 @@ SELECT
   version,
   GET_LIGHTHOUSE(ARRAY_AGG(STRUCT(
     client,
-    median_lighthouse_score_accessibility,
-    median_lighthouse_score_best_practices,
-    median_lighthouse_score_performance,
-    median_lighthouse_score_pwa,
-    median_lighthouse_score_seo
+    median_lighthouse_score.accessibility,
+    median_lighthouse_score.best_practices,
+    median_lighthouse_score.performance,
+    median_lighthouse_score.pwa,
```

Comment:
LEt's drop PWA. Lighthouse doesn't report it anymore and although it may have some historical interest, I don't think it's enough to warrant keeping it around.